### PR TITLE
Fixed URL, former domain is now for sale

### PR DESCRIPTION
**Pull request by nicolas-raoul**
New URL found in the description fields at https://github.com/reduxframework/redux-framework
- minor grammar fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # react-redux-material_ui-boilerplate
-A boilerplate for React + Redux + Material UI + ES6 syntax applications. This boilerplate includes following tools and frameworks.
+A boilerplate for React + Redux + Material UI + ES6 syntax applications. This boilerplate includes the following tools and frameworks:
 
 * [React](https://facebook.github.io/react/)
-* [Redux](http://rackt.org/redux/index.html)
+* [Redux](https://reduxframework.com/)
```
Comment (JLHwung) [inline README.md]: Redux framework is a Wordpress framework, see http://redux.js.org

**Review by JLHwung — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (takanabe) [reply README.md]: @nicolas-raoul  please modify url to the correct one, then I will merge this PR.

**Comment by nicolas-raoul** [comment]
Sorry for the delay, is it better now?

**Comment by takanabe** [comment]
Thanks!

**Pull request merged**